scripts/fetch_config/fetch_config.rb

```diff
@@ -333,14 +351,18 @@ def open_in_editor(config_map)
   output_string = config_map.to_s
 end
 
+
+##### Confirm #####
+ask_to_confirm(destination, output_string) if confirm
+
 ##### Output #####
 case destination[:type]
 when 'stdout'
   puts output_string
 when 'file'
-  File.write(destination[:locator], output_string)
```

Comment:
Can we add an output message similar to the other destination path cases, maybe something like this:
```
when 'file'
  puts "Writing to #{destination[:parameter]}"
  File.write(destination[:parameter], output_string)
...
```
Just to confirm the action has taken place, what do you think?

Reply:
The confirmation will happen in this case as well if you ask for it (-c). You can also use -v for verbose and get more feedback.

Reply:
Also, I'd rather not add non essential output by default as `stdout` should stay raw so it can be printed, dumped to file or piped to another command's stdin.
I'll merge for now, we can discuss improvements later on.

Reply:
that's fair 👍